demo/README.md

```diff
@@ -59,25 +57,30 @@ $ sudo docker rmi <image-id>
 $ sudo docker system prune -a
 ```
 
-# Start Manufacturer Service
+# Configuring Proxies
+Update the proxy information in `_JAVA_OPTIONS` as
 
-Refer [Demo Manufacturer README](manufacturer/README.md) to start the service.
+```
+_JAVA_OPTIONS=-Dhttp.proxyHost=http_proxy_host -Dhttp.proxyPort=http_proxy_port -Dhttps.proxyHost=https_proxy_host -Dhttps.proxyPort=https_proxy_port
+```
+
+where
 
-# Start Reseller Service
+`http_proxy_host`: Represents the http proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
 
-Refer [Demo Reseller README](reseller/README.md) to start the service.
+`http_proxy_port`: Represents the http proxy port. Typically, it is the port number in the proxy URL.
 
-# Start Rendezvous Service
+`https_proxy_host`: Represents the https proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
 
-Refer [Demo Rendezvous README](rv/README.md) to start the service.
+`https_proxy_port`: Represents the https proxy port. Typically, it is the port number in the proxy URL.
 
-# Start Owner Service
+Specify combination the hostname and port information together for either http, https, or both. For example, if the http proxy is 'http://myproxy.com:900', then the following updates will be made to the properties:
```

Comment:
Specify the combination of the hostname and the port information together ...

Reply:
Done